README.md

```
@@ -308,6 +240,46 @@ automatically—no manual submission required.
+
+**[📋 Take the Survey →](https://tally.so/r/q4Ed88)**
+
+---
+
+## 💬 Discord
+
+You can submit any feedback in our [discord server](https://discord.gg/NBW4zeFcG6)
+
+<p align="center">
+  <a href="https://discord.gg/NBW4zeFcG6">
+    <img src="https://invidget.switchblade.xyz/NBW4zeFcG6" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)